packages/core/src/messagingProtocols/mls/MLSService/MLSService.ts

```diff
@@ -97,6 +97,7 @@ export class MLSService extends TypedEventEmitter<Events> {
       nbKeyPackages = defaultConfig.nbKeyPackages,
       cipherSuite = defaultConfig.cipherSuite,
     }: Partial<MLSServiceConfig>,
+    private readonly e2eServiceExternal: E2EIServiceExternal,
```

Comment:
as a convention we usually pass service early in the constructor's parameters. at least it should be before the config